tutorials/ardu-pi-serial-part-1/index.md

```diff
@@ -0,0 +1,330 @@
+
+### Install Cloud SDK and dependencies for Pi Sense HAT
+1. SSH to the Raspberry Pi with: `$ ssh pi@<your_Pi_IP>`
+1. On the Pi, follow all the steps [here][cloudsdk] to install and initialize Cloud SDK for Debian systems.
+1. Check that Cloud SDK is installed and initialized with: `$ gcloud info`. Ensure that the `Account` and `Project` properties are set correctly.
+1. Install Git, Python development packages and Sense HAT dependencies with:
+```
+$ sudo apt-get update && sudo apt-get install -y git virtualenv python-dev sense-hat libffi-dev libssl-dev libopenjp2-7 libjpeg-dev
+```
+
+### Clone the Pi client app and install its dependencies
+1. Using the SSH connection to the Pi, clone the repo associated with the community tutorials:
+```
+$ git clone https://github.com/GoogleCloudPlatform/community.git
+```
+1. Change to the tutorial directory:
+```
+$ cd community/tutorials/ardu-pi-serial/part1
+```
+1. Create a Python virtual environment:
+```
+$ virtualenv venv
+```
+1. Activate the virtualenv:
+```
+$ source venv/bin/activate
+```
+1. Upgrade pip and setuptools:
+```
+(venv) $ pip install -U pip setuptools
+```
+1. Install the required Python modules:
+```
+(venv) $ pip install -r requirements.txt
+```
+1. Copy the private key file that you generated from IoT Core [Quickstart][quickstart] guide to the app’s current working directory on the Pi. Back on your laptop command line, you can use `scp` to transfer the file to the Pi, with for example:
+```
+$ scp rsa_private.pem pi@<your_Pi_IP>:ardu-pi-serial/
+```
+You can check your Pi's IP address by executing: `$ ifconfig` on the Pi.
+
+[cloudsdk]: https://cloud.google.com/sdk/docs/#deb
+
+## Arduino Uno installation and configuration
+### Connect the sensors to the Arduino
+
+**Figure 3.** *IoT boards and components*
+![boards diagram][boardsdiag]
+[boardsdiag]: ardu-pi.png
+
+1. Connect 2 analog sensors to the Arduino. See [Figure 2][boardsdiag] to see how to wire the 2 sensors to the Arduino. In this example, we will read the analog voltage values of a DC motor, and a solar panel. *Note: you can replace the DC motor and solart panel with other analog sensors - but you **have to ensure that the sensor output does not exceed 5V** or it can damage the Arduino pin.*
+1. Connect the first sensor (small DC motor) to the Arduino. Connect the black ground wire of the motor to a `GND` port, and the red (voltage input/output) wire to port `A0` on the Arduino.
+1. Connect the second sensor (small solar panel) to the Arduino. Connect the black ground wire of the panel to a `GND` port, and the red (voltage output) wire to port `A1` on the Arduino.
+
+### Compile the Arduino Sketch
+1. Connect your Arduino to your local development machine with a USB cable. Note that the Arduino Uno board used as the example in this tutorial, has a USB type B connector on the board. The USB cable will provide both power, and serial data connectivity to the Arduino from your machine.
```

Comment:
Strike the comma after the word "tutorial", as well as the comma after the word "power".

Reply:
Done